system/modules/core/library/Contao/Files/Php.php

```diff
@@ -57,6 +57,9 @@ public function rmdir($strDirectory)
 	 */
 	public function fopen($strFile, $strMode)
 	{
+		if (is_link($strFile)) {
+			$strFile = readlink($strFile);
```

Comment:
```TL_ROOT``` is prepended in the second next line, so prepending it here would cause double prepending.

I even think it should NOT be prepended anyhow, if the path is a symbolic link, as the target could be outside of ```TL_ROOT```.